### ISSUE TITLE
[StratCon] GoodEvent.xml not doing anything?

### ISSUE THREAD

**Issue by Tecmes** [MegaMek/mekhq#4097]
49.19-snap, story arc version

The GoodEvent.xml, added when your side controls a Comm Center, seems to do nothing, at least nothing obvious, and the content of the XML doesn't *seem* to do much (but I can't say I'm positive about that).

**Comment by Thom293** [comment]
Yes.  Bad event was broken, and this works the same way, so I am sure it is too.  I "fixed" bad event by changing the malus, and I can do it here too, but it changes its effects.  Ill try to look at next time I have some time.
